Scripts/DCS-BIOS/lib/modules/aircraft_modules/Bf-109K-4.lua

```diff
@@ -0,0 +1,350 @@
+Bf_109K_4:defineFloat("AFN2_HORIZONTAL_NEEDLE", 14, { -1, 1 }, "Indicator", "AFN2_Horizontal_Needle")
+Bf_109K_4:defineFloat("AFN2_VERTICAL_NEEDLE", 15, { 0, 1 }, "Indicator", "AFN2_Vertical_Needle")
+Bf_109K_4:defineIndicatorLight("AFN2_SIGNAL_LAMP", 16, "Indicator", "AFN2_Signal_Lamp (white)")
+Bf_109K_4:defineFloat("FUEL_PRESSURE", 25, { 0, 1 }, "Indicator", "Fuel_Pressure")
+Bf_109K_4:defineFloat("OIL_PRESSURE", 26, { 0, 1.0 }, "Indicator", "Oil_Pressure")
+Bf_109K_4:defineFloat("COOLANT_TEMPERATURE", 27, { -1, 1 }, "Indicator", "Coolant_Temperature")
+Bf_109K_4:defineFloat("OIL_TEMPERATURE", 28, { -1, 1 }, "Indicator", "Oil_Temperature")
+Bf_109K_4:defineFloat("ENGINE_RPM", 29, { 0, 1 }, "Indicator", "Engine_RPM")
+Bf_109K_4:defineFloat("PROPELLER_PITCH_HOUR", 30, { 0, 1 }, "Indicator", "Propeller_Pitch_Hour")
+Bf_109K_4:defineFloat("PROPELLER_PITCH_MINUTE", 31, { 0, 1 }, "Indicator", "Propeller_Pitch_Minute")
+Bf_109K_4:defineFloat("MANIFOLD_PRESSURE", 32, { 0, 1 }, "Indicator", "Manifold_Pressure")
+Bf_109K_4:defineFloat("FUEL_TANK_FUSELAGE", 33, { -1, 1 }, "Indicator", "Fuel_Tank_Fuselage")
+Bf_109K_4:defineFloat("OXYGEN_PRESSURE", 34, { 0, 1 }, "Indicator", "Oxygen_Pressure")
+Bf_109K_4:defineFloat("OXYGEN_FLOW_BLINKER", 35, { 0, 1 }, "Indicator", "Oxygen_Flow_Blinker")
+Bf_109K_4:defineFloat("VARIOMETERNEEDLE", 36, { -0.3, 0.3 }, "Indicator", "VariometerNeedle")
+Bf_109K_4:defineFloat("MG131_0_SHELL_COUNTER", 40, { 0, 1 }, "Indicator", "MG131_0_Shell_Counter")
+Bf_109K_4:defineFloat("MG131_1_SHELL_COUNTER", 41, { 0, 1 }, "Indicator", "MG131_1_Shell_Counter")
+Bf_109K_4:defineFloat("MG131_0_KLAPPANKER", 42, { 0, 1 }, "Indicator", "MG131_0_Klappanker")
+Bf_109K_4:defineFloat("MG131_1_KLAPPANKER", 43, { 0, 1 }, "Indicator", "MG131_1_Klappanker")
+Bf_109K_4:defineIndicatorLight("LEFTWING_MG151_CONTROL_LAMP", 44, "Indicator", "LeftWing MG151 Control Lamp (yellow)")
+Bf_109K_4:defineIndicatorLight("RIGHTWING_MG151_CONTROL_LAMP", 45, "Indicator", "RightWing MG151 Control Lamp (yellow)")
+Bf_109K_4:defineFloat("BODYSTATE", 46, { 0, 1 }, "Indicator", "BodyState")
+Bf_109K_4:defineFloat("SMOKEDSCREEN", 47, { 0, 1 }, "Indicator", "SmokedScreen")
+Bf_109K_4:defineFloat("SIGHTBRIGHTNESS", 48, { 0, 1 }, "Indicator", "SightBrightness")
+Bf_109K_4:defineIndicatorLight("LANDINGGEARREDLIGHT", 56, "Indicator", "LandingGearLight (red)")
+Bf_109K_4:defineIndicatorLight("LANDINGGEARGREENLIGHTLEFT", 57, "Indicator", "LandingGearLightLeft (green)")
+Bf_109K_4:defineIndicatorLight("LANDINGGEARGREENLIGHTRIGHT", 58, "Indicator", "LandingGearLightRight (green)")
+Bf_109K_4:defineIndicatorLight("FUEL_RESERVE_LAMP", 164, "Indicator", "Fuel Reserve Lamp")
+
+-- Gauge Formulas
+local function getManifold()
+	local returnValue = (GetDevice(0):get_argument_value(32)) * 65 + 10
+	return returnValue
+end
+Bf_109K_4:defineIntegerFromGetter("MANIFOLD_PRESSURE_VALUE", getManifold, 65000, "Gauge Values", "Manifold Pressure kg/cm2")
```

Comment:
You should be able to use module:defineGaugeValue to clean up all these defineIntegerFromGetter calls

Reply:
OK!

Reply:
This is the only change. I assume it is OK. Since 65535 is specified as max value in the defineGaugeValue().

![bk109](https://github.com/DCS-Skunkworks/dcs-bios/assets/10453261/063030e2-698c-40af-8abc-53d333aafbba)